pkg/coscheduling/coscheduling_test.go

```diff
@@ -193,10 +226,10 @@ func TestLess(t *testing.T) {
 			expected: true, // p1 should be ahead of p2 in the queue
 		},
 		{
-			name: "equal priority. p1 is added to schedulingQ earlier than p2, p1 belongs to podGroup1",
+			name: "equal priority. p1 is added to schedulingQ earlier than p2, p1 belongs to podGroup3",
 			p1: &framework.QueuedPodInfo{
 				Pod: &v1.Pod{
```

Comment:
I know this is pre-existing. Could you help to refactor a bit:

```
Pod: st.MakePod().Name("pod1").Namespace("namespace1").Label(pgutil.PodGroupLabel, "pg3").Priority(highPriority),
```

So that we can remove labels1 to labels4.

Reply:
done